### DISCUSSION TITLE
Help with maintaining this repo?

### DISCUSSION THREAD

**Discussion by dwradcliffe** [jlvaillant/intellicenter#63] [General]
Hi @jlvaillant,
Thanks again for a great integration. I know life gets busy sometimes - I have seasons too. I'd love to offer to help maintain this repo to share the burden if you're willing. Let me know.
-David

**Comment by philipflesher** [comment]
Likewise. I have just recently made some changes to add better support for IntelliChlor and would like to get them reviewed and merged. I have designs on fixing my other recently submitted issue about long-term statistics support.

Can I help on repo maintenance/ownership going forward?

**Reply by kracklow** [reply]
Is this integration still being maintained? I am starting worry I may need to block updates to firmware on my Intellicenter in order to maintain compatibility.

**Comment by philipflesher** [comment]
@dwradcliffe @kracklow given the lack of responses, I'm considering a fork and getting some fixes/updates in over the coming months. I would prefer to share the burden on this as I don't have time to fully maintain on my own, and of course the more diverse equipment we're testing against, the better the integration will be. Any interest in assisting?

To any other folks who stumble by and want to help: the more the merrier, from my perspective.

**Reply by TheRydad** [reply]
My deep hands on programming experience (with the exception of some light C# for playing with Arduino) is 20+ years old. I actually have been reading through the code here over the past couple of years. While I am basically able to follow what is going on, I need to brush up on some fundamentals.

I just completed a commitment that was my primary time commitment, so I now have some free time to retrain myself.

Bottom line, I'd love to help, but I am a ways away from being of any use.

Note: if anyone has any tips on where to get some Python tutorials, I'd appreciate it. I am sure a quick search will get me going, though.

**Comment by kracklow** [comment]
I am not a coder but I can test, I have intellicenter, intellchor and pentair gas heater along with a valve for heater flow, also using some AUX circuits to integrate landscape and pool lighting into HomeAssistant.

**Comment by dwradcliffe** [comment]
For anyone following along I'm planning on only using [my fork](https://github.com/dwradcliffe/intellicenter) moving forward. I'll do my best to accept PRs there.

**Comment by kracklow** [comment]
Thanks @dwradcliffe will start watching your repo. My pool in in hibernation until early April most likely, but I have Intellicenter, Intelliclor, and a Pentair Gas heater along with one programmable valve.

**Reply by kracklow** [reply]
@dwradcliffe  What is the best way to switch to your fork? Do I need to remove it from HA first?

**Reply by dwradcliffe** [reply]
Don't remove the device from HA. First remove the integration from HACS. Do not restart. Add my repo to HACS. Then when you restart it should just load up with the new code.

**Comment by CryptoCouyon** [comment]
@dwradcliffe I may have missed it, as I am not very proficient in GitHub. Is there any plan to be able to adjust the variable-speed drive's speed/flow settings from the integration?

**Reply by CryptoCouyon** [reply]
And thank you so much for this integration! It is absolutely wonderful to simplify my pool operations for my family!

**Reply by dwradcliffe** [reply]
There’s some discussion here https://github.com/jlvaillant/intellicenter/discussions/12#discussioncomment-711242
For now the workaround is using feature circuits.
